sdk/storage/azure-storage-blob/README.md

```diff
@@ -187,6 +187,54 @@ async for blob in container.list_blobs():
 print(blob_list)
 ```
 
+## Optional Configuration
+
+Optional keyword arguments that can be passed in at the client and per-operation level. 
+
+### Retry Policy configuration
+
+Use the following keyword arguments when instantiating a client to configure the retry policy:
+
+* _retry_total_ (int): Total number of retries to allow. Takes precedence over other counts.
+    Pass in `retry_total=0` if you do not want to retry on requests. Defaults to 10.
+* _retry_connect_ (int): How many connection-related errors to retry on. Defaults to 3.
+* _retry_read_ (int): How many times to retry on read errors. Defaults to 3.
+* _retry_status_ (int): How many times to retry on bad status codes. Defaults to 3.
+* _retry_to_secondary_ (bool): Whether the request should be retried to secondary, if able.
+    This should only be enabled of RA-GRS accounts are used and potentially stale data can be handled.
+    Defaults to False.
```

Comment:
We also support `retry_policy` to pass in a custom instance of `~azure.storage.blob.ExponentialRetry` or `azure.storage.blob.LinearRetry`. The default configuration uses an `ExponentialRetry`.

Reply:
This reminds me..I believe we don't want to allow passing a retry_policy. Thoughts @johanste

Reply:
Confirmed with @johanste . We don't want to document it now. We'll eventually support passing a string instead of an instance,